scripts/env-setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+################################################
+# Generates and sets environment variables for broker
+#
+################################################
+
+# Set the network in the .env file based on user input
+echo "Enter the network: 1 for MainNet, 2 for TestNet"
+read ADDR
+if [ $ADDR = '1' ]; then
+  sed -i "" -e "s/.*NETWORK.*/NETWORK=mainnet/" .env
+elif [ $ADDR = '2' ]; then
+  sed -i "" -e "/NETWORK/c\NETWORK=testnet" .env
+else
+  echo "$ADDR is not a valid option for network"
+  exit 1
+fi
+
+# Set the external btc/ltc addresses in the .env file based on user public IP address
+echo "Enter your public IP address:"
+read IP_ADDRESS
+
+sed -i "" -e "s/^EXTERNAL_BTC_ADDRESS.*/EXTERNAL_BTC_ADDRESS=$IP_ADDRESS/" .env
+sed -i "" -e "s/^EXTERNAL_LTC_ADDRESS.*/EXTERNAL_LTC_ADDRESS=$IP_ADDRESS/" .env
+
+array=(BTC_RPC_USER
+BTC_RPC_PASS
+LTC_RPC_USER
+LTC_RPC_PASS
+RPC_USER
```

Comment:
do we want to randomly generate the RPC user and pass as well?

How do we expect the user to get those values into their CLI config?

Reply:
We tell them to get it from their .env file in the instructions. It seemed less complicated to do it this way, than to have them manually add anything to the .env file. Thoughts?